src/modules/queue/components/Queue.jsx

```diff
@@ -6,101 +6,147 @@ import YoutubeResult from '../../uniplayer/Search/YoutubeResult';
 import queue_img from './images/queue.svg';
 import x_img from './images/x.svg';
 
-export default function Queue({isQueueOpen, isSearchOpen, queue, toggleQueue, toggleSearch}) {
-	if (isQueueOpen) {
-		return (
-			<Sidebar expand className="sidebar">
-				<h2>Tracklist</h2>
-				<Search />
-				<QueueList>{queue.map(track => <YoutubeResult result={track.content} key={track.id} highRes />)}</QueueList>
-				<QueueButton onClick={toggleQueue} open>
-					<img src={x_img} />
-				</QueueButton>
-			</Sidebar>
-		);
-	}
+export default function Queue({isSearchOpen, queue, toggleSearch}) {
 	return (
-		<Sidebar className="sidebar">
-			<QueueButton onClick={toggleQueue}>
-				<OButton />
-			</QueueButton>
-		</Sidebar>
+		<QueueOuter>
+			<QueueInner expand className="queue">
+				<QueueHeader>
+					<CollectionRecord>
+						<span />
+					</CollectionRecord>
+					<CollectionInfo>
+						<h5>COLLECTION</h5>
+						<h1>/example</h1>
+						<AddSong onClick={toggleSearch}>
+							Add Song
+							<span> +</span>
+						</AddSong>
+					</CollectionInfo>
+					{!isSearchOpen && <h4>{queue.length} tracks</h4>}
+					{isSearchOpen && <h4>Search for a song on YouTube or Soundcloud</h4>}
+				</QueueHeader>
+				<QueueList>{queue.map(track => <YoutubeResult result={track.content} key={track.id} highRes />)}</QueueList>
```

Comment:
I'm never actually seeing the queue, it's always behind the search (I think because it has a fixed height)